testdata/invalidlineage/compat/remove-optional.txtar

```diff
@@ -29,4 +29,6 @@ lin: lenses: [{
 	}
 }]
 -- out/bindfail --
-schema 0.1 must be backwards compatible with schema 0.0
+schema 0.1 is not backwards compatible with schema 0.0:
+field not allowed in closed struct: getsRemoved
```

Comment:
Here I guess we'll also need to pay attention when working on messages clarity, as closedness is probably something not explicitly managed by the schema autor, even less by a non-author user.